torch/csrc/cudnn/Conv.cpp

```diff
@@ -362,8 +362,19 @@ Convolution::Convolution(
     params.pad[i] = pad[i];
     params.stride[i] = stride[i];
     params.dilation[i] = dilation[i];
+    CHECK_ARG(stride[i] > 0);
+    CHECK_ARG(dilation[i] > 0);
   }
   params.groups = groups;
+
+  for (int i = 2; i != input->nDimension; ++i) {
+    int kernel_size = params.weight_size[i];
```

Comment:
params.weight_size[i] will always be greater than 0 right? (because tensor dims are never < 0).
What you want to check is that params.size[i] > 0 yea?